overlord/configstate/config/helpers.go

```diff
@@ -291,6 +291,7 @@ type Conf interface {
 	Get(snapName, key string, result interface{}) error
```

Comment:
I considered in the past moving this interface, maybe it's time to move it to configcore as it's the only place using it, together also with ConfGetter perhaps?

Reply:
I've pushed a commit doing this now.